lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -235,21 +268,42 @@ class RenderPopover extends RenderShiftedBox {
     markNeedsPaint();
   }
 
+  /// Returns the size which must be reserved to display the arrow.
```

Comment:
There are multiple word choices that make this comment a bit confusing.

"size which must be reserved" - this is stated as if someone is going to implement this behavior, and you're telling them what they need to implement. But that's not the case here, is it? We've already implemented the relevant behavior. Therefore we should write comments in terms of "what is" rather than "what must be done".

"the [child] can take the space that is reserved for the arrow" - this is phrased like a choice, but it isn't a choice, is it? If `extentAndClipContentOverArrow` is `true` then the [child] **will** extend in all four directions and will be clipped by the bar and arrow, right?

"When [allowHorizontalArrow] is `false`, the reserved width is always zero." - I'm not sure what this is trying to say.